testing/runner.ts

```diff
@@ -1,44 +1,42 @@
 #!/usr/bin/env -S deno -A
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 import { parse } from "../flags/mod.ts";
-import { glob, isGlob, walk } from "../fs/mod.ts";
-import { runTests } from "./mod.ts";
-const { args, cwd } = Deno;
-
-const DEFAULT_GLOBS = [
-  "**/*_test.ts",
-  "**/*_test.js",
-  "**/test.ts",
-  "**/test.js"
-];
-
-/* eslint-disable max-len */
+import {
+  WalkInfo,
+  expandGlobSync,
+  glob,
+  ExpandGlobOptions
+} from "../fs/mod.ts";
+import { isWindows } from "../fs/path/constants.ts";
+import { isAbsolute, join } from "../fs/path/mod.ts";
+import { RunTestsOptions, runTests } from "./mod.ts";
+const { DenoError, ErrorKind, args, cwd, exit } = Deno;
+
+const DIR_GLOBS = [join("**", "?(*_)test.{js,ts}")];
+
 function showHelp(): void {
   console.log(`Deno test runner
 
 USAGE:
-  deno -A https://deno.land/std/testing/runner.ts [OPTIONS] [FILES...]
+  deno -A https://deno.land/std/testing/runner.ts [OPTIONS] [MODULES...]
 
 OPTIONS:
-  -q, --quiet               Don't show output from test cases 
-  -f, --failfast            Stop test suite on first error
-  -e, --exclude <FILES...>  List of file names to exclude from run. If this options is 
-                            used files to match must be specified after "--". 
-  
-ARGS:
-  [FILES...]  List of file names to run. Defaults to: ${DEFAULT_GLOBS.join(
-    ","
-  )} 
-`);
-}
-/* eslint-enable max-len */
+  -q, --quiet                 Don't show output from test cases
+  -f, --failfast              Stop running tests on first error
+  -e, --exclude <MODULES...>  List of comma-separated modules to exclude
+  --allow-none                Exit with status 0 even when no test modules are
+                              found
 
-function filePathToRegExp(str: string): RegExp {
-  if (isGlob(str)) {
-    return glob(str, { flags: "g" });
-  }
-
-  return RegExp(str, "g");
+ARGS:
+  [MODULES...]  List of test modules to run.
+                A directory <dir> will expand to:
+                  ${DIR_GLOBS.map((s: string): string => `${join("<dir>", s)}`)
+                    .join(`
+                  `)}
+                Defaults to "." when none are provided.
+
+Note that modules can refer to file paths or URLs and globs are supported for
+the former.`);
```

Comment:
```
Deno test runner

USAGE:
  deno -A https://deno.land/std/testing/runner.ts [OPTIONS] [MODULES...]

OPTIONS:
  -q, --quiet                 Don't show output from test cases
  -f, --failfast              Stop running tests on first error
  -e, --exclude <MODULES...>  List of comma-separated modules to exclude
  --allow-none                Exit with status 0 even when no test modules are
                              found

ARGS:
  [MODULES...]  List of test modules to run.
                A directory <dir> will expand to:
                  <dir>/**/?(*_)test.{js,ts}
                Defaults to "." when none are provided.

Note that modules can refer to file paths or URLs and globs are supported for
the former.
```

Nice :+1: 

Maybe the last sentence could be changed to `Note that modules can refer to file paths or URLs. File paths support glob expansion.`

Reply:
Also I wouldn't mind if there were 1-2 examples at the end of help message

Reply:
👍